### DISCUSSION TITLE
[targetname] doesn't support translation overrides

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14773] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

A bit of a weird one but:
If a medic NPC tries to heal Rex (or any creature in the "human" group), their text will use speciesname instead of the speciestranslationoverride

In this case, instead of saying "Mudraptor" (or Rex), they say "Mudraptor_Passive"

**`<dialogfoundwoundedtarget>Giving first aid to [targetname] in [roomname]!</dialogfoundwoundedtarget>`**

![image](https://github.com/user-attachments/assets/43363cbe-bf60-4548-907e-1a2edf7f7a5f)

 ```xml
 <Charactervariant inherit="Mudraptor" speciesname="Mudraptor_passive" group="human" speciestranslationoverride="Mudraptor" texture="Content/Characters/Mudraptor/Mudraptor.png">
 ````
 
 Modded:
![image](https://github.com/user-attachments/assets/9d978913-d05f-4ee2-b615-378761803b6e)

### Reproduction steps

1. Load an outpost
2. Trigger event
3. Shoot raptor
4. NPCs try to heal raptor

```xml
    <ScriptedEvent identifier="targetnameTEST">
      <TagAction criteria="player" tag="player" />
      <SpawnAction speciesname="mudraptor_passive" spawnpointtag="player" targettag="test"/>
    </ScriptedEvent> 
```

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
[name] on the other hand properly uses the overridden name

```xml
<dialogcannottreatpatient>I don't have the means to treat [name]!</dialogcannottreatpatient>
````

![image](https://github.com/user-attachments/assets/bfcbde61-9cef-4e3f-85a4-771a9f80aa71)

**Comment by Regalis11** [comment]
Fixed in our private repo now, closing
